### ISSUE TITLE
Unlocalized strings in PySolFC 2.14.0

### ISSUE THREAD

**Issue by Lakrits** [shlomif/PySolFC#233]
**Unlocalized strings in PySolFC 2.14.0**

I have found a fair collection of unlocalized strings in PySolFC (v.2.14.0). Since I have thirteen screenshots, and all would need an accompanying description, this issue entry would be a bit too large to be readable. For that reason, I've included the screenshots (PNG's), and a TXT file with descriptions in the ZIP file below. All screenshots have the unlocalized strings marked with red rectangles.

[PySolFC 2.14.0 unlocalized strings.zip](https://github.com/shlomif/PySolFC/files/7670833/PySolFC.2.14.0.unlocalized.strings.zip)

**Comment by joeraz** [comment]
A lot of those reported strings are from more recent features, and I don't believe anyone's currently doing the translations.  I can easily set up the files to input the localized strings, but I'd need the translations to complete this.

I could just use Google Translate, that's not always super accurate, but I'm considering it.

**Comment by Lakrits** [comment]
By "unlocalized", I don't mean untranslated. I mean, that these strings haven't been made available for translation in the POT file pysol.pot. The screenshots are from my own personal translation of PySolFC into Swedish. The strings you see, and which I have reported, are those that were left over when the sv_pysol.po file had been fully translated.

Attaching the file sv_pysol.po for good measure.
[sv_pysol.zip](https://github.com/shlomif/PySolFC/files/7674542/sv_pysol.zip)

**Comment by joeraz** [comment]
I see what you're saying.  I'm working on updating the POT file for the strings used in the upcoming 2.14.1 release, though I'll have to hold on adding any translated values unless I can get someone to assist with the actual translations.

However, a few of the strings you listed do not come from PySol - as these options could potentially change based on individual configurations, I won't be adding these for the moment:
- The table tiles are based on the image files located in the data/tiles folder.
- The toolbar styles are based on the subfolders located in the data/images/toolbar folder.

Thanks.

**Comment by Lakrits** [comment]
Thank you for the answer.

As for my translation, it's just for my family, to make it easier for them to use the programme (excellent programme, I might add). You don't have to add it to the repository, unless you want to use it as a template for future Swedish translators of the programme. I added the file just to make it possible for you to see that it was fully translated (which, in hindsight, was rather unnecessary, I admit).

**Comment by joeraz** [comment]
An updated POT has been added to the code to be released with 2.14.1 (most likely tomorrow).  This should contain most of the strings you pointed out, and a few others I found (though a few are related to features that were designed for the next major release and won't be in 2.14.1 - I'll update the main repo with these once 2.14.1 is live).

If you'd like me to include your translation file, feel free to send me a copy or make a PR for it when it's finished with all the newest scripts.  Otherwise, I'll leave it be.  Just note that many of the features I am working on will involve more new strings being added in the next couple big releases.
